nixos/modules/services/databases/postgresql.xml

```diff
@@ -63,25 +53,83 @@ Type "help" for help.
   <title>Upgrading</title>
 
   <para>
-   FIXME: document dump/upgrade/load cycle.
+   Major PostgreSQL upgrade requires PostgreSQL downtime and a few imperative steps to be called. To simplify this process, use the following NixOS module:
```

Comment:
You may want to link the upstream docs about upgrading (if any) here. This provides users an easy way to troubleshoot in case something goes wrong.

And it might be useful to remind people reading this to make a backup first :)